proto/info/patch_envelope.proto

```diff
@@ -61,3 +61,22 @@ message ZInfoOpaqueAppInstanceStatus {
     EveOpaqueStatusAction opaqueStatusAction = 3;
     bytes opaqueStatus = 4;
 }
+
+// Provide information on PatchEnvelope usage per app
+message ZInfoPatchEnvelopeStat {
+    // count the number of times app instance called patch APIs
+    uint64 patchApiCallCount = 1;
+    // information about every patchEnvelope referred to this app
+    repeated ZInfoPatchEnvelopeApp patchStatus = 2;
+}
+
+// Statistics on usage of specific patchEnvelope for specific app
+message ZInfoPatchEnvelopeApp {
+    // UUID of PatchEnvelope
+    string uuid = 1;
+    // Version of PatchEnvelope
+    string version = 2;
+    // Number of times this PatchEnvelope
+    // was sucessfully downloaded
+    uint64 timesDownloaded = 3;
```

Comment:
What did we decide regarding updates to a patch envelope? Will the controller update it and use the same UUID? Or generate a new UUID for the updated PE?

The download counter makes some sense in the former case.
Is this the number of times the app instance has retrieved the PE? 

Makes sense adding some more comments to make the intended semantics and usage more clear.

Reply:
We do the former, that's why it makes sense